locales/en/yourloops.json

```diff
@@ -110,6 +112,9 @@
   "chat-messages-header": "Messages",
   "choice-of-healthcare-professional": "Choice of healthcare professional",
   "choice-of-remote-monitoring-team": "Choice of remote monitoring team",
+  "coefficient-of-variation": "CV ({{cbgLabel}})",
+  "coefficient-of-variation-empty-stat": "**Why is this stat empty?**\n\nThere is not enough data present in this view to calculate it",
```

Comment:
This translation exists twice in this file. Refactor it under a more generic key name